server/src/test/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStoreTests.java

```diff
@@ -450,5 +468,24 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         }
     }
 
+    public class TestClock extends Clock {
+        public Queue<Instant> instants = new LinkedList<Instant>();
+
+        @Override
+        public ZoneId getZone() {
+            return ZoneId.systemDefault();
```

Comment:
Using the system default zone wouldn't be good in some situations, but probably works fine for this test.